set-integration-testing/src/integration-test/java/eu/europeana/api/set/integration/web/SearchUserSetRestIT.java

```diff
@@ -299,7 +299,8 @@ public void searchSetByTextQueryDefault() throws Exception {
             .param(CommonApiConstants.QUERY_PARAM_PROFILE, LdProfiles.STANDARD.name())
             .queryParam(CommonApiConstants.PARAM_WSKEY, API_KEY)
             .queryParam(CommonApiConstants.QUERY_PARAM_QUERY, query)
-            .queryParam(CommonApiConstants.QUERY_PARAM_PAGE_SIZE, PAGE_SIZE))
+            .queryParam(CommonApiConstants.QUERY_PARAM_PAGE_SIZE, PAGE_SIZE)
+            .queryParam(CommonApiConstants.QUERY_PARAM_SORT, WebUserSetFields.TEXT_SCORE_SORT))
```

Comment:
the sorting by score with asc and desc order needs to be tested in own test and 2 set are needed to check that the sorting+ordering works correctly

Reply:
The score sort supports only the desc order per default, and cannot be asc.

Reply:
the sort is a user input and the validation of the user input needs to be applied, Consiquently the api behaviour needs to be covered by integration tests as well